charts/rancher-logging/0.1.2/Chart.yaml

```diff
@@ -6,7 +6,8 @@ version: 0.1.2
 appVersion: 1.6.3
 home: https://www.fluentd.org/
 sources:
-- https://www.fluentd.org/
+  - https://www.fluentd.org/
```

Comment:
Not sure whats going on with all these formatting, but as long as it compiles as valid yaml it should be fine.

Reply:
it is valid yaml so this should be fine.